sonar-db/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -731,16 +735,16 @@ public void updateBEnabledToFalse() {
     assertThat(row2.get("bQualifier")).isEqualTo(dto2.qualifier());
 
     Map<String, Object> row3 = selectBColumnsForUuid("U3");
-    assertThat(row3.get("bChanged")).isIn(false, /* for Oracle */ 0L, 0);
+    assertThat(row3.get("bChanged")).isIn(false, /* for Oracle */0L, 0);
   }
 
   private Map<String, Object> selectBColumnsForUuid(String uuid) {
     return db.selectFirst(
-        "select b_changed as \"bChanged\", b_copy_component_uuid as \"bCopyComponentUuid\", b_description as \"bDescription\", " +
-          "b_enabled as \"bEnabled\", b_language as \"bLanguage\", b_long_name as \"bLongName\"," +
-          "b_module_uuid as \"bModuleUuid\", b_module_uuid_path as \"bModuleUuidPath\", b_name as \"bName\", " +
-          "b_path as \"bPath\", b_qualifier as \"bQualifier\" " +
-          "from projects where uuid='" + uuid + "'");
+      "select b_changed as \"bChanged\", b_copy_component_uuid as \"bCopyComponentUuid\", b_description as \"bDescription\", " +
```

Comment:
one shall not alter production code for test purpose only => I won't change the mapper.